read-fonts/src/tables/aat.rs

```diff
@@ -373,40 +375,51 @@ impl<'a> SomeTable<'a> for StateTable<'a> {
     }
 }
 
-pub struct ExtendedStateTable<'a, T = ()> {
-    header: StxHeader<'a>,
+#[derive(Clone)]
+pub struct ExtendedStateTable<'a, T = NoPayload> {
```

Comment:
Looks like a nice opportunity to add some docs 😏

Reply:
Done!